### PR TITLE
Fix/popover remove style

### PR DESCRIPTION
**Pull request by chrismanciero**
### Description
Popover component was applying a CSS class (example: `fd-popover--right`) that does not exist in the Fiori Fundamental CSS Style guide. This PR removes that applying of the class to the Popover component wrapper div.

### CODE REVIEW

File: src/Popover/Popover.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import classnames from 'classnames';
-import {POPOVER_ALIGNMENTS} from '../utils/constants';
+import { POPOVER_ALIGNMENTS } from '../utils/constants';
```
Comment (bcullman) [inline src/Popover/Popover.js]: unnecessary whitespace change. If you prefer this style, please open a separate PR, with a lint rule enforcing this style

```
object-curly-spacing: [2, always]
```

Comment (chrismanciero) [reply src/Popover/Popover.js]: The shift again happened from some VS Code setting on a TypeScript extension that I removed. Will look into why it's being applied or will add the lint rule.
